packages/client/components/ui/components/design/layout/Column.stories.tsx

```diff
@@ -48,7 +48,7 @@ export default {
         grow: true,
       },
       decorators: [
-        (props) => (
+        (props: any) => (
```

Comment:
Im going to let this slide _just because_ i have no clue what type `props` should be

Reply:
I don't think it has a type

Reply:
huh

i guess i'll check it later